package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@keep-network/tbtc.js",
-  "version": "0.19.4-pre",
+  "version": "0.19.4-rc",
```

Comment:
This doesn't match `package.json`.

Reply:
Fixed, see b1eeab0af4ffb3ae4118d73cd27c8ee6676ba5ab.